content/shmem_init_thread.tex

```diff
@@ -0,0 +1,48 @@
+\apisummary{
+Initializes the \openshmem{} library, similar to \FUNC{SHMEM\_INIT}, and in addition
+perform the initialization required for thread-safe invocation of \openshmem{} functions.}
```

Comment:
I see your point. In addition to your proposed sentence, how about we specifically say that initialization required for supporting four thread levels. Something like this 
`Initializes the OpenSHMEM library, similar to shmem_init, and performs any initialization required for supporting four thread levels`